docs/2022/project/updates/2022-06-02.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Community Bonding
+title: Week 1
```

Comment:
We consider 1st week of coding period as week 1. So please consider calling it Community Bonding

Reply:
@fwhdzh , can you please fix it?

Reply:
Hi, I have make a new commit to solve this. Thank you!